BigBaseV2/src/services/context_menu_service.cpp

```diff
@@ -2,6 +2,7 @@
 #include "natives.hpp"
 #include "pointers.hpp"
 #include "gta/replay.hpp"
+#include <gui.hpp>
```

Comment:
🤔 `<>`

Reply:
Ah, I know why now, Intellisense default to this when you use for auto resolve missing import

Reply:
Bad defaults 😏